packages/bezier-react/src/components/Tooltip/Tooltip.module.scss

```
@@ -0,0 +1,37 @@
+@use "../../styles/mixins/typography";
+
+@layer components {
+  .tooltip-content {
```

```suggestion
  .Tooltip {
```

아직 명확한 컨벤션은 없으나, module의 root class는 컴포넌트명이랑 통일하면 좋을 거 같아요!

---

SegmentedControl 마이그레이션해보니까.. 아래와 같이 네이밍 규칙을 정하면 어떨까 싶네요

- 컴포넌트에 적용되는 root class는 파스칼 케이스로 한다
- 상태에 따른 nested class는 케밥 케이스로 한다
  - 처음에 `styles.fooStyle` 같은 식으로 사용하기 편해서 카멜 케이스를 고려해봤는데, `styles[display-${display}]` 같은 식으로 케밥 케이스가 케이스 변환없이 사용하기 편한 장점이 있네요. #1850 이슈를 고려해보면 카멜 케이스가 좋을 거 같은데(computed key를 사용하지 않는 방향), 나중에 적용해도 괜찮지 않을까.. 싶었습니다

---

넵 저도 동의합니다!